tree/dataframe/src/RDataFrame.cxx

```diff
@@ -694,6 +708,16 @@ RDataFrame d("myTree", "file.root");
 std::vector<std::string> colNames = d.GetColumnNames();
 ~~~
 
+### Organizing RDataFrame in collections
+
+Any RDataFrame node can be converted to the common type ROOT::RDF::RNode. The cast can happen either implicitly by the C++ casting rules or explicitly using ROOT::RDF::AsRNode. This allows to organize RDataFrame nodes in any collection, e.g. a std::vector<ROOT::RDF::RNode>. Note that this adds an extra virtual call during the RDataFrame event loop any have an impact on the performance.
```

Comment:
In C++ there is no need for `AsRNode`, you can just construct an `RNode` from any RDF object:
```suggestion
Any RDataFrame node can be converted to the common type ROOT::RDF::RNode. The cast can happen either implicitly by the C++ casting rules or by passing any dataframe object to RNode's constructor. This allows to organize RDataFrame nodes in any collection, e.g. a `std::vector<ROOT::RDF::RNode>`. Note that this adds an extra virtual call during the RDataFrame event loop and can have a minor performance impact.
```